virtualLightSensor/virtualLightSensor-ES6.yaml

```diff
@@ -121,10 +121,16 @@ actions:
 
 
         function itemState(event, name) {
+          try {
             // if the event was triggered by the item, make sure to return the event state
             if (event && (event.itemName === name))
-                return event.itemState.toString();
-            return items.getItem(name).state;
+                return Quantity(event.itemState.toString());
+            return items[name].quantityState;
```

Comment:
I doubt this will work if using the Okta binding for cloudiness values. The okta cloudiness is a Number from 0 to 8, not a QauntityType.

Reply:
It works. 

If you create a `Quantity` with a plain number without units, it treates it as a plain number without units (e.g. `Quantity(event.itemState.toString()).symbol` would be `null`). However, if the `event.itemState.toString()` is `"5"`, `Quantity(event.itemState.toString()).int` would be `5`.

Reply:
But it fails in the next line (so not with the event, but when retrieving directly from the item).